discovery-provider/src/models.py

```diff
@@ -879,3 +879,90 @@ def __repr__(self):
 txhash={self.txhash},\
 created_at={self.created_at},\
 updated_at={self.updated_at})>"
+
+class ChallengeType(str, enum.Enum):
+    boolean = 'boolean'
+    numeric = 'numeric'
+
+class Challenge(Base):
+    __tablename__ = "challenges"
+
+    id = Column(String, primary_key=True, nullable=False, index=True)
```

Comment:
can we comment on what each of these fields is for (particularly step count, that's the most interesting one)